### PR TITLE
same fix to process creation done on qc

### PR DESCRIPTION
**Pull request by UlysseFG**

### CODE REVIEW

File: backend/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-3.1.0
+3.2.0
```
Comment (jleblond) [inline backend/VERSION:1]: I wouldnt change the version to 3.2 already.. those modifications are still part of 3.1.1 just to be clear and it's in QC too

**Review by jleblond — APPROVED** [review]
Apart from my comment on the version number, all good!

File: backend/VERSION (hunk 1 above)
Comment (UlysseFG) [reply backend/VERSION:1]: This is master... it won't make its way to prod before 3.2... It was mainly not to forget (i tend to forget to update this).

Comment (jleblond) [reply backend/VERSION:1]: right, my suggestion was mainly to keep the commits more organized, but since it's not a critical issue, that works for me too

**Pull request merged**